libusb.go

```diff
@@ -41,6 +42,12 @@ type libusbDevHandle C.libusb_device_handle
 type libusbTransfer C.struct_libusb_transfer
 type libusbEndpoint C.struct_libusb_endpoint_descriptor
 
+type libusbOpt int
+
+const (
+	LIBUSB_OPTION_NO_DEVICE_DISCOVERY libusbOpt = iota
```

Comment:
I don't believe you've addressed the user-visible portion of the API, essentially passing the options to NewContext()

I'd suggest:

```
type ContextOptions struct {
  DisableDeviceDiscovery bool
}

func (o *ContextOptions) NewContext() (*Context, error) { ... }
```